src/Exceptions/HttpException.php

```diff
@@ -73,6 +88,24 @@ public static function buildJsonResponse(): JsonResponse
         $phrase = static::PHRASE;
         $href = static::HREF;
 
-        return new JsonResponse(compact("code", "description", "phrase", "href"), static::CODE);
+        if (!strlen($phrase)) {
+            throw new RuntimeException("Phrase constant is not defined.");
+        }
+
+        if (!strlen($description)) {
+            throw new RuntimeException("Description constant defined.");
+        }
+
+        $json = compact("description", "phrase");
+
+        if ($code) {
+            $json["code"] = $code;
+        }
+
+        if (strlen($href)) {
+            $json["href"] = $href;
+        }
+
+        return new JsonResponse($json, static::CODE);
     }
 }
```

Comment:
File should have a unix line ending it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces